docs/essentials/pipeline-queues.md

```diff
@@ -0,0 +1,197 @@
+---
+description: See how to assign pipelines to execution queues and run them
+sequentially or how to run them in parallel to save time.
+---
+
+# Pipeline Queues
+
+Pipeline queues allow you to control which pipelines Semaphore must run
+sequentially and which may run in parallel. For example, you may configure
+parallel pipelines on the main branch, while allowing only one deployment to
+production to be running at any given time.
+
+You can configure this behaviour via [queue][queue-reference] property in the
+pipeline's YAML configuration file.
+
+## Default behaviour
+
+By default, the pipelines will be assigned to the same queue and run sequentially  
+if they are initiated from the same branch/tag/pull request with the same
+configuration, e.g. multiple pushes or multiple promotions.
+
+## Using queues for deployments
+
+You can use queues to prevent parallel runs of deployment pipelines that might
+cause issues in the environment you are deploying to.
+
+If your projects are deployed independently one from another, you can use the
+separate queues for each project by setting the `scope` property of the queue
+configuration to value **project**.
+
+In the following example, we are configuring the deployment pipeline that should
+be defined as a [promotion][deploying-with-promotions] to run sequentially in
```

Comment:
Good idea to link to a concept here, but you mention promotions earlier, always link to the first ever mention.

Reply:
This link needs to be in this section to indicate that the deployment should be implemented as a promotion.

I moved it to the first mention of the `deployment pipelines` at the beginning of this section and that should accomplish both things.